### DISCUSSION TITLE
Welcome to wcag2ict Discussions!

### DISCUSSION THREAD

**Discussion by michael-n-cooper** [w3c/wcag2ict#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maryjom** [comment]
Thanks @michael-n-cooper for setting this up. As someone astutely pointed out in the meeting, most of the WCAG2ICT participants have their GitHub ID showing up in the [participant's page](https://www.w3.org/groups/tf/wcag2ict/participants). There are a few people's IDs that don't show there, and I'm not sure why. Is this because they haven't linked their GitHub ID to their W3C ID? Is that the final step? 

For example, Laura's GitHub id is lboniello, but it doesn't show on the WCAG2ICT participants page.

**Reply by shawnthompson** [reply]
They would have to go to [W3C User Account Management](https://www.w3.org/Help/Account/), then click on "Edit your account information". Once in you will have a link on the left side that says "Connected Accounts". 

That's where they can connect their Github account to their W3C account information.

![image](https://user-images.githubusercontent.com/2575266/196784596-20e1e486-f33e-496e-a31d-8e01ffe64ba4.png)

alt: image showing the list of links on the W3C members profile page with the "Connected Accounts" link.

**Reply by maryjom** [reply]
Thanks @shawnthompson. I had emailed them instructions for connecting their accounts and all are showing up correctly now.

**Comment by bruce-usab** [comment]
Three mostly unrelated questions / comments:
- [x] The current [issues](../issues) do not look much specific to WCAG2ICT.  Are they seeds from the WCAG repo?
- [ ] ~~Can we break the document into chucks?   So maybe more like the [WCAG2 repo](../../wcag)?~~ OBE
- [ ]  ~~Can we please please please give folks the option to write using Markdown?~~  OBE

**Reply by shawnthompson** [reply]
+1 on switching to markdown for the documentation. 

One of the problems we found when with using Markdown to edit files. There is no editor format toolbar like there is with issues and discussions.

![image](https://user-images.githubusercontent.com/2575266/196974262-81bd2caf-ffe4-4273-8ffc-7fa375c39967.png)

**Reply by shawnthompson** [reply]
Looks like we can use Markdown with respec.

[Using Markdown with ReSpec](https://github.com/w3c/respec/wiki/markdown)

**Reply by maryjom** [reply]
@shawnthompson That's good news. @michael-n-cooper @daniel-montalvo If either of you can show me what to do, I can get the transformation started. Seems pretty easy.  Just change the ReSpec config to include: 
```
var respecConfig = {
  format: "markdown"
}
```
Then make separate files out of each section, left justify all code, and adjust the HTML markup to use markdown instead. Or we could leave in the existing HTML since it still works alongside the markdown.

I would need to learn how to do the including of the separate .md files to get the Note built again.

**Comment by bruce-usab** [comment]
@shawnthompson that is an example of embedding Markdown inside of HTML -- which can have some utility, but is not the main benefit to having a Markdown oriented workflow.

For example, the backend of [Section508.gov](https://www.section508.gov) is a layperson friendly [GitHub directory](https://github.com/GSA/Section508.gov/tree/main/_pages)

A Markdown oriented repo not only provides easier syntax but also:
1. Content broken down into chucks
2. Built-in preview functionality.

**Reply by shawnthompson** [reply]
@bruce-usab I completely agree with you. You folks are using Jekyll to build your webpages but W3C is using ReSpec which doesn't build from `.md` files.

After speaking with @michael-n-cooper we said we could switch to use [bikeshed](https://github.com/tabatkins/bikeshed) but it's a bit of a learning curve for him. 

I really think it would be a benefit to switch to markdown (`.md`) files and showing folks how to use the editor in the browser to make changes instead of pushing, pulling and branching. The Github interface does it all for them, even if they didn't have access.

It would be nice if Github could offer the markdown toolbar when editing a file though. I wonder if they are working on an implementation already... hmmmmmm

**Reply by bruce-usab** [reply]
I have on-boarded several average-tech colleagues.

The **lack** of a toolbar is a plus!  I need content contributors to pay attention to structure.  Plain text (for writing/editing) has been so good.  Markdown for emphasis, headings, lists, links, etc. is lightweight.  I have not had any trouble teaching that part at all.

GitHub generates a good-enough preview.  If the casual end-user spots a typo, they just edit again.

Most of the .gov Jekyll repos have Netlify, which defaults to providing a WYSIWYG interface for writing/editing articles.  That feature of Netlify has actually been counter-productive!

**Reply by maryjom** [reply]
Wonder how easily you can reproduce the special formatting for Editors' notes, etc. using markdown. I have used markdown in the past, but nothing too fancy.

**Reply by bruce-usab** [reply]
Markdown allows reference to CSS classes.  One adds `{: .classname }`

An example on the Section 508 site is:  https://github.com/GSA/Section508.gov/blob/main/_pages/acquisition/2018-05-02-buy-sell.md and the `{: .text-gray-50 }` after the first paragraph.

The public facing version of that source code:  https://www.section508.gov/buy-sell/

That page source is mostly HTML, but there is still a human-friendly preview via GitHub and not just raw source code.

**Reply by maryjom** [reply]
Moving this conversation to a new discussion to continue the conversation.

**Comment by ChrisLoiselle** [comment]
This is my comment about discussions within the discussions area , testing 1, 2, 3.
